Sources/Orbit/Components/Tag.swift

```diff
@@ -9,8 +9,8 @@ import SwiftUI
 /// - Note: [Orbit definition](https://orbit.kiwi/components/tag/)
 public struct Tag: View {
 
-    public static let height = Spacing.xLarge
-    public static let minWidth = Spacing.xLarge
+    public static let verticalPadding: CGFloat = 7
```

Comment:
Do we really need all these constants to be `public`?

Reply:
Apart from more complicated alignment guides, this is the way to make clients be able to align their custom content next to Orbit content (so that they do not have to "guess" those paddings)